### ISSUE TITLE
Not maked as fork

### ISSUE THREAD

**Issue by GiulianoWF** [freakontrol/stmbl#1]
Not an actual issue, but your repository isn't marked as a fork of the original project.

I recently checked out the original repo, and it seems to be inactive. While searching for good forks, I first looked at the official forks list, but couldn't find any that seams to active. But I saw in the linux form your version that seams nice.

Maybe if you could mantain a clone of this repo that's actually maked as a fork of the original, or opening an issue in the original pointing to here. More people could see this repo.

Any way, nice work!

**Comment by freakontrol** [comment]
Hi, yes I knew that, the problem was Github doesn't allow [LFS](https://github.com/git-lfs/git-lfs/issues/1906) in a repo fork, so I had to create a "fork" manually and not using Github. After I also realized that there are strict limitations in bandwidth when using LFS and I had to disable it again or anybody couldn't fetch some files. After all this, I think the best thing to do is as you suggested creating an issue in the original repo. If you wish you're free to do it.
